packages/volto/src/components/manage/Blocks/Video/Body.jsx

```diff
@@ -94,52 +121,41 @@ const Body = ({ data, isEditMode }) => {
             'full-width': data.align === 'full',
           })}
         >
-          {data.url.match('youtu') ? (
+          {hasMatch ? (
             <>
-              {data.url.match('list') ? (
-                <Embed
-                  url={`https://www.youtube.com/embed/videoseries?list=${listID}`}
-                  {...embedSettings}
-                />
-              ) : (
-                <Embed id={videoID} source="youtube" {...embedSettings} />
-              )}
+              <Embed {...embedSettings} />
             </>
+          ) : Object.keys(iframeSettings).length > 0 ? (
+            <iframe title="Peertube video iframe" {...iframeSettings}></iframe>
```

Comment:
Why can't we use `Embed` for Peertube?

Reply:
Because we're not sure how to get a correct placeholder for each video.